activesupport/lib/active_support/testing/method_call_assertions.rb

```diff
@@ -30,6 +30,11 @@ def assert_called_with(object, method_name, args = [], returns: nil)
         def assert_not_called(object, method_name, message = nil, &block)
           assert_called(object, method_name, message, times: 0, &block)
         end
+
+        def stub_any_instance(klass, instance = klass.new)
```

Comment:
I could see us make the method an argument in the future (so it's something else than `new`). But we shouldn't do that now, unless you can show me cases where that would make sense.

Anyway, we should make `instance` a keyword argument.